### PR TITLE
<SectionHelper /> - change scss to stylable

### PR DESCRIPTION
**Pull request by sivanharel**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5697/

File: src/SectionHelper/SectionHelper.js [hunk 1]
```diff
@@ -77,11 +60,7 @@ class SectionHelper extends React.PureComponent {
           </div>
         )}
 
-        <div
-          className={classNames(styles.content, {
-            [styles.fullWidth]: !!fullWidth,
-          })}
-        >
+        <div {...styles('content', { fullWidth: !!fullWidth })}>
```
Comment (TalZa1810) [inline src/SectionHelper/SectionHelper.js:63]: fullWidth is boolean already so `{ fullWidth  }` is fine.
Moreover, I think that this state can be moved to the root div.

File: src/SectionHelper/SectionHelper.js [hunk 2]
```diff
@@ -41,16 +30,10 @@ class SectionHelper extends React.PureComponent {
...
-        })}
+        {...styles('root', { appearance, withCloseBtn: showCloseButton }, {})}
       >
         {showCloseButton && onClose && (
-          <div
-            className={classNames(styles.close, {
-              [styles.closeWithTitle]: !!title,
-            })}
-          >
+          <div {...styles('close', { withTitle: !!title })}>
```
Comment (TalZa1810) [inline src/SectionHelper/SectionHelper.js:36]: move the state to the root div

**Comment by sivanharel** [comment]
The types test is failing since I changed the PropTypes without changing the index.d.ts file.
I don't really need to change this file since the only change I did was to change the appearance prototype to be hardcoded in order to fix the API in the storybook.

**Pull request merged**